car-service/src/main/kotlin/com/curious/programmers/carservice/CarServiceImpl.kt

```diff
@@ -0,0 +1,18 @@
+package com.curious.programmers.carservice
+
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+@Component
```

Comment:
Yep, I know, but... As it's a service, shouldn't we `@Service` annotation as well?